packages/create/templates/hello-prisma/start_postgres_docker.js

```diff
@@ -18,7 +18,7 @@ if (!process.env.PGPASSWORD) {
 }
 
 try {
-  execSync(`docker run --rm --name=dbos-db --env=POSTGRES_PASSWORD="${process.env.PGPASSWORD}" --env=PGDATA=/var/lib/postgresql/data --volume=/var/lib/postgresql/data -p ${port}:5432 -d postgres:16.1`);
+  execSync(`docker run --rm --name=dbos-db --env=POSTGRES_PASSWORD="${process.env.PGPASSWORD}" --env=PGDATA=/var/lib/postgresql/data --volume=/var/lib/postgresql/data -p ${port}:5432 -d sibedge/postgres-plv8`);
```

Comment:
Have you confirmed this is Postgres 16?

Reply:
Yes. The [Dockerfile](https://github.com/sibedge-llc/plv8-dockerfiles/blob/master/alpine/Dockerfile) for this image builds off of `postgres:16.3-alpine`